addons/stock_account/models/template_generic_coa.py

```diff
@@ -0,0 +1,24 @@
+from odoo import models
+from odoo.addons.account.models.chart_template import template
+
+
+class AccountChartTemplate(models.AbstractModel):
+    _inherit = "account.chart.template"
+
+    @template('generic_coa', 'res.company')
+    def _get_generic_coa_res_company(self):
+        res = super()._get_generic_coa_res_company()
+        res[self.env.company.id].update({
+            'account_production_wip_account_id': 'wip',
+            'account_production_wip_overhead_account_id': 'cost_of_production',
+        })
+        return res
```

Comment:
You could write a separate `@template` method that returns just the extra values for the company. Then you could just reference that in the post_init script.
Usually, we don't inherit `@template` methods (even though in this case it's technically fine) as the `@template` registers the method in a list of methods to call to get the template data.

```suggestion
    @template('generic_coa', 'res.company')
    def _get_generic_coa_wip_accounts(self):
        return {self.env.company.id: {
            'account_production_wip_account_id': 'wip',
            'account_production_wip_overhead_account_id': 'cost_of_production',
        }}
```

Reply:
I'll check if the original author did it this way on purpose for some reason, but we'll change it if it wasn't needed, thanks!